### PR TITLE
docs: updated the PMT docs structure

### PR DESCRIPTION
**Pull request by alexandra-mara**
Solves #101

### CODE REVIEW

**Comment by NickVolynkin** [comment]
# Table of contents

* [`=nil;` Proof Market](README.md)
  `+=` [Supported proof systems](getting-started/supported-proof-systems.md)

## Getting started

* [Sign up](user-guides/sign-up.md) `+=` part 1 of Web interface

## Proof Market

* [Market economics](market/economics.md)
* [Market mechanics](market/mechanics.md)



## User guides and tutorials

* [Web interface](market/web-interface.md)




## Proof Market Toolchain

* [Get the toolchain](getting-started/installation.md)
  `+=` [Environment setup](getting-started/environment-setup.md) → Building from source → Link to readme?
* [Zero-knowledge app developer](user-guides/zk-app-developer.md)
* [Proof producer](user-guides/proof-producer.md)
* [Requester/producer flow](user-guides/requester-producer-flow.md)
* [Reference](market/cli-reference/README.md)
  * [User account utilities](market/cli-reference/user.md)
  * [Statement utilities](market/cli-reference/statement.md)
  * [Proposal utilities](market/cli-reference/request.md)
  * [Request utilities](market/cli-reference/proposal.md)
  * [Proof utilities](market/cli-reference/proof.md)




## Examples

* [Mina proof pipeline example](examples/mina/README.md)
  * [Part 1. Setup](examples/mina/part-1-setup.md)
  * [Part 2. Deploying zkApp](examples/mina/part-2-deploy-zkapp.md)
  * [Part 3. Requesting a proof](examples/mina/part-3-proof-request.md)
  * [Part 4. Proof validation](examples/mina/part-4-validate-proof.md)

## Misc

* ~[Contributing](misc/contributing.md)~
* ~[Code of conduct](misc/code-of-conduct.md)~
* ~[Contact us](misc/contact.md)~ — goes

**Comment by alexandra-mara** [comment]
@NickVolynkin here's my suggestion. We can't just move the toolchain part to just zk app developers section, as it's intended for all market users. So let's have it as a section?

# Table of contents
* [`=nil;` Proof Market](README.md)

## Getting started
* [Starting as a developer](toolchain/sign-up.md)
* [Starting as a proof producer](toolchain/sign-up.md#proof-producer)

## Proof Market overview
* [Market economics](market/economics.md)
* [Market mechanics](market/mechanics.md)
* [Web interface](market/web-interface.md)

## Proof Market toolchain
* [Toolchain installation](toolchain/installation.md)

* [Reference](toolchain/cli-reference/README.md)
  * [Account utilities](toolchain/cli-reference/user.md)
  * [Statement utilities](toolchain/cli-reference/statement.md)
  * [Proposal utilities](toolchain/cli-reference/request.md)
  * [Request utilities](toolchain/cli-reference/proposal.md)
  * [Proof utilities](toolchain/cli-reference/proof.md)

## zk applications developers
* [Starting as a developer](toolchain/sign-up.md)
* [Zero-knowledge app developer](developers/zk-app-developer.md)

## Proof producers
* [Starting as a proof producer](toolchain/sign-up.md#proof-producer)
* [Proof producers](producers/proof-producer.md)

## zk applications examples
* [Interacting with Proof Market toolchain](examples/toolchain.md)
* [Mina zk app + Proof Market](examples/mina-example/README.md)
  * [Part 1. Setup](examples/mina-example/part-1-setup.md)
  * [Part 2. Deploying a zk app](examples/mina-example/part-2-deploy-zkapp.md)
  * [Part 3. Requesting a proof](examples/mina-example/part-3-proof-request.md)
  * [Part 4. Proof validation](examples/mina-example/part-4-validate-proof.md)

## Misc
* [Contact us](misc/contact.md)

**Comment by NickVolynkin** [comment]
```
* Getting started
* Proof Market
* Application developers
* Proof producers
* Application case study (examples)
* Reference
```

File: gitbook-docs/SUMMARY.md [hunk 1]
```diff
@@ -2,41 +2,42 @@
...
+* [Toolchain installation](toolchain/installation.md)
+* [Zero-knowledge app developer guide](developers/zk-app-developer.md)
+
+## Proof producers
+
+* [Getting started](toolchain/sign-up.md#proof-producer)
+* [Toolchain installation](toolchain/installation.md)
+* [Proof producers guide](producers/proof-producer.md)
+
+## Application case study (examples)
```
Comment (NickVolynkin) [inline gitbook-docs/SUMMARY.md:23]: Let's just say Application examples

Comment (alexandra-mara) [reply gitbook-docs/SUMMARY.md:23]: Fixed

**Review by NickVolynkin — APPROVED** [review]

**Comment by NickVolynkin** [comment]
Testing:

- [ ] https://docs.nil.foundation/proof-market/user-guides/proof-requester
- [ ] https://docs.nil.foundation/proof-market/user-guides/proof-producer

**Comment by alexandra-mara** [comment]
Redirects fixed in #119

**Comment by alexandra-mara** [comment]
@NickVolynkin the links were wrong. Redirects work :)
For example:
https://docs.nil.foundation/proof-market/market/user-guides/proof-requester
https://docs.nil.foundation/proof-market/misc/contributing

**Pull request merged**